.github/workflows/build_python_package.yml

```diff
@@ -36,19 +36,8 @@ jobs:
       - name: Set environment variable
         id: vars
         run: |
-          if [[ -z "${{ inputs.module-name}}" ]]
-          then
-            echo "MODULE_PATH=code" >> $GITHUB_ENV
-          else
-            echo "MODULE_PATH=code/${{ inputs.module-name }}" >> $GITHUB_ENV
-          fi
-
-          if [[ -z "${{ inputs.package-name}}" ]]
-          then
-            echo "PACKAGE_NAME=${{ inputs.module-name }}" >> $GITHUB_ENV
-          else
-            echo "PACKAGE_NAME=${{ inputs.package-name }}" >> $GITHUB_ENV
-          fi
+          echo "MODULE_PATH=code/${{ inputs.package-path }}" >> $GITHUB_ENV
```

Comment:
Remove step and use arguments directly.

Reply:
The 'code/' string before inputs.package-path should be removed and specified by the file that invoke this one?